src/registration/de_DE.json

```diff
@@ -67,5 +68,5 @@
   "MANUAL": "Anleitung",
   "YOUR_SENSEBOX": "Deine senseBox",
 
-  "SAME_ADDRESS_WARNING": "Sie wollen mehrere Sensoren mit der gleichen Adresse nutzen! Dafür wird ein Multiplexer benötigt."
+  "SAME_ADDRESS_WARNING": "Sie wollen mehrere Sensoren mit der gleichen Adresse nutzen! Dafür wird ein Multiplexer benötigt.",
```

Comment:
The `,` is not allowed on last line or no following property and is blocking the tests.